tests/filename_map.yml

```
@@ -57,9 +57,9 @@ salt/modules/influxdb08mod.py:
   - pytests.unit.states.test_influxdb08_user
 
 salt/modules/mysql.py:
-  - unit.states.test_mysql_user
-  - unit.states.test_mysql_query
-  - unit.states.test_mysql_grants
+  - pytests.unit.states.mysql.test_mysql_user
+  - pytests.unit.states.mysql.test_mysql_query
+  - pytests.unit.states.mysql.test_mysql_grants
```

```suggestion
  - pytests.unit.states.mysql.test_user
  - pytests.unit.states.mysql.test_query
  - pytests.unit.states.mysql.test_grants
```
And also rename the test modules, there's no need to have `mysql` in the path twice.

---

Fixed